### PR TITLE
Settings redux redo

### PR DESCRIPTION
**Pull request by epoww**
Settings -> Account Screen
Added redux for home section ordering and bookmarked articles
Styles to match Discover Screen

### CODE REVIEW

File: src/screens/SettingsScreen.js [hunk 1]
```diff
@@ -54,30 +87,56 @@ const styles = StyleSheet.create({
 const settings_sections = [
```
Comment (peterbaile) [inline src/screens/SettingsScreen.js]: this should probably go into `constants.js`

File: src/screens/ManageFeedScreen.js [hunk 2]
```diff
@@ -64,85 +63,101 @@ const styles = StyleSheet.create({
   },
 })
 
-class ManageFeedScreenComp extends Component {
+class ManageFeedForPublication extends Component {
```
Comment (peterbaile) [inline src/screens/ManageFeedScreen.js:66]: turn into functional component

File: src/screens/ManageFeedScreen.js [hunk 3]
```diff
@@ -64,85 +63,101 @@ const styles = StyleSheet.create({
...
 
-  _renderRow = ({ data, active }) => {
-    return <Row data={data} active={active} />
-  }
+const ManageFeedScreenComp = ({
+  navigation,
+  publication,
+  settings,
+  dispatch,
+}) => {
```
Comment (peterbaile) [inline src/screens/ManageFeedScreen.js]: omit unnecessary `{}`, just say `=> (<ManageFeedForPublication>...)`

File: src/screens/ManageFeedScreen.js [hunk 4]
```diff
@@ -64,85 +63,101 @@ const styles = StyleSheet.create({
...
+      key={publication}
+      navigation={navigation}
+      publication={publication}
+      homeSectionPreferences={settings.homeSectionPreferences}
+      dispatch={dispatch}
+    />
+  )
 }
 
 class Row extends Component {
```
Comment (peterbaile) [inline src/screens/ManageFeedScreen.js:162]: convert to functional component

**Review by peterbaile — CHANGES_REQUESTED** [review]
good job, fix the coding suggestions i made and we r gtg

File: src/screens/ManageFeedScreen.js (hunk 4 above)
Comment (epoww) [reply src/screens/ManageFeedScreen.js:162]: going to leave this - will replace entire component in different PR

File: src/screens/ManageFeedScreen.js (hunk 2 above)
Comment (epoww) [reply src/screens/ManageFeedScreen.js:66]: going to leave this - will replace entire component in different PR

File: src/screens/ManageFeedScreen.js (hunk 3 above)
Comment (epoww) [reply src/screens/ManageFeedScreen.js]: done

File: src/screens/SettingsScreen.js (hunk 1 above)
Comment (epoww) [reply src/screens/SettingsScreen.js]: done

**Pull request merged**